recipes/stages/detection/semisl.py

```diff
@@ -1,9 +1,13 @@
 _base_ = [
-    './finetune.py',
+    './train.py',
+    '../_base_/data/coco_ubt.py',
+    '../_base_/models/detectors/detector.py'
 ]
 
-model = dict(
-    unlabeled_loss_weight=1.0,
+task_adapt = dict(
+    type='mpa',
+    op='REPLACE',
+    efficient_mode=False,
```

Comment:
Is False default option for detection?

Reply:
I guess so, I copied from class-incr recipe